converged_sound_open_fw_modules_development/FW/intel_common/module_binmaps/aca_module.binmap

```diff
@@ -0,0 +1,35 @@
+module o ACA
```

Comment:
ok, so IIUC we have a 1:1 mapping between binmap file and module ?

Reply:
Yes, between aca_module and aca_module.binmap files.